docs/Audio.md

```diff
@@ -29,12 +29,62 @@ Determines if a referenced track is currently playing. Requires a valid referenc
 
 Ends the currently playing BGM. Will restart if resumed.
 
+Also exists with an "Optional Fade" variant, which requires a fade duration (in sec.)
+
 ### Suspend BGM
 
 Suspends the currently playing BGM. If resumed, will continue from the time it was suspended.
 
+Also exists with an "Optional Fade" variant, which requires a fade duration (in sec.)
+
 ### Resume BGM
 
+Will resume the music where it left off, with a fade in. The fade duration is set in the BGM controller's parameters.
+
+Also exists with an "Optional Fade" variant, which requires a fade duration (in sec.) which will override the BGM controller's.
+
+### Get Musical Time
+
+Given a BGM controller, gets the play time of its currently BGM track. The play time's value ("Musical time") is not in seconds, and does not directly scale with real-time values.
+
+Requires a valid reference to a `SQEXSEADBGMSLot Controller` object.
+
+#### Converting Musical Time values into real-time
+
+Each BGM controller has a (hard-coded?) range of value for the musical time. This range is determined by the duration of the unmodded BGM track that the BGM controller calls:
+- BGM track has a loop:
+	- Musical time range = -1 to (unmodded BGM track length)/2
+- BGM track doesn't have a loop:
+	- Musical time range = 0 to (unmodded BGM track length)/2
+
+These values **don't** change even when replacing the original BGM track.
```

Comment:
This *sounds* like you can't change where a track loops when replacing audio? Am I reading this right?

Reply:
Not quite, you can replace any looped BGM track with a new BGM with a different loop, but the BGM controller will continue to tick Musical Time values based on the unmodded track. I'll update the page!